system-addon/content-src/styles/_activity-stream.scss

```diff
@@ -22,12 +21,7 @@ h2 {
 }
 
 a {
-  color: $link-primary;
   text-decoration: none;
-
-  &:hover {
-    color: $link-secondary;
```

Comment:
Removing the :hover -secondary is right, so we can get rid of this overriding color too:
https://github.com/mozilla/activity-stream/blob/837cea5f88ceac381a95b5c6bdb0f1e3106d2ad2/system-addon/content-src/components/TopSites/_TopSites.scss#L364-L366

I seem to recall design wanting links to underline on hover.. but I guess leave it this way for now.